src/pages/Home.tsx

```diff
@@ -21,6 +21,11 @@ export default function Home() {
   /* Retrieve user data from server*/
   useEffect(() => {
     const getUserData = async () => {
+      const response = await axios(`/api/me`, {
```

Comment:
is this supposed to be here? this is the same api call as in line 30 inside the if statement